packages/docs/pages/dropdown.tsx

```diff
@@ -11,348 +11,450 @@ import {
+                    It is possible to add a <Code>description</Code> for items.
+                  </Text>
 
-  return (
-    <>
-      <H1>Dropdown</H1>
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    <Dropdown
+                      items={[
+                        {
+                          hash: '1',
+                          content: 'Item #1',
+                          description: 'Description for option #1',
+                          onItemClick: (item) => item,
+                        },
+                        {
+                          content: 'Item #2',
+                          description: 'Description for item #2',
+                          disabled: true,
+                          type: 'link',
+                          url: '#',
+                        },
+                        { hash: '3', content: 'Item #3', onItemClick: (item) => item },
+                        { hash: '4', content: 'Item #4', onItemClick: (item) => item },
+                        { hash: '5', content: 'Item #5', onItemClick: (item) => item },
+                      ]}
+                      toggle={<Button>Button</Button>}
+                    />
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+          ]}
+        />
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <ContentRoutingTabs
+          id="props"
+          routes={[
+            {
+              id: 'dropdown',
+              title: 'Dropdown',
+              render: () => <DropdownPropTable renderPanel={false} />,
+            },
+            {
+              id: 'dropdown-item',
+              title: 'DropdownItem',
+              render: () => <DropdownItemPropTable renderPanel={false} />,
+            },
+            {
+              id: 'dropdown-link-item',
+              title: 'DropdownLinkItem',
+              render: () => <DropdownLinkItemPropTable renderPanel={false} />,
+            },
+            {
+              id: 'dropdown-item-group',
+              title: 'DropdownItemGroup',
+              render: () => <DropdownItemGroupPropTable renderPanel={false} />,
+            },
+          ]}
+        />
+      </Panel>
 
-      <PageNavigation items={items} />
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={[
+            'Anchor the dropdown menu in a position with is least obstructive to page information.',
+            'Orgainze dropdown items in alphabetical order or by the most relevant content.',
+            'Use placeholder text by default if no selection has been made.',
+          ]}
+          discouraged={[
+            'Avoid including complex information in a dropdown menu.',
+            <>
+              Don’t nest <Code primary>Dropdowns</Code> or use them to display complex information.
```

Comment:
Should we omit this `use them to display complex information` (since line `450` already mentioned that) and just leave `Don't nest Dropdowns`.

Reply:
That would work from a writing standpoint.